src/Ng/Jobs/LoopingNgJob.cs

```diff
@@ -38,8 +38,15 @@ protected LoopingNgJob(ILoggerFactory loggerFactory)
                 }
 
                 Logger.LogInformation("Running job.");
-                await RunInternal(cancellationToken);
-                Logger.LogInformation("Job finished!");
+                try
+                {
+                    await RunInternal(cancellationToken);
+                    Logger.LogInformation("Job finished!");
+                }
+                catch (TransientException te)
+                {
+                    Logger.LogWarning("Job execution ended due to a transient exception: {0}. {1}", te.Message, te.InnerException?.StackTrace??te.StackTrace);
```

Comment:
Something I thought about this weekend...

The way you've written this, the job will still wait the entire interval before trying again. If we know we encountered an intermittent failure, wouldn't we want to try again immediately rather than wait?

The interval of the V3 jobs is low (30 seconds) but I'm slightly concerned about waiting when we know the failure was intermittent.